README.md

```diff
@@ -1,117 +1,175 @@
+<a name="readme-top"></a>
 
-<!-- TABLE OF CONTENTS -->
+<div align="center">
+  <br/>
+
+  <h3><b>Portfolio: application deploy</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTSs -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
+    - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
 - [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 [Portfolio Desktop-Version] <a name="about-project"></a>
 
-# 📖 [About-Myself] <a name=About-Myself></a>
+The aim of this project is to build a simple personal portfolio web page while understanding how to parse a Figma design and flexbox to create a UI and usse the images and backgrounds to enhance the look of the website and also to use Media queiry condition to make website responsive fot both mobile and desktop.
 
 
-**[About-Myself]** is a simple project built with html and css showing responsiveness for mobile version. It showa my accomplishments and skills as a developer.
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML & CSS 
+- Linter
+- Git, Github and Visual Studio Code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🛠 Built With
-1. Html
-2. Css
-3. Linters
+### Tech Stack <a name="tech-stack"></a>
 
-## Tech Stack <a name="tech stack"></a>
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://html.org/">Html</a></li>
-    <li><a href="https://css.org/">Css</a></li>
+    <li><a href="https://git-scm.com/">Git</a></li>
   </ul>
 </details>
 
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Template</summary>
+  <ul>
+    <li><a href="https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=2tXla7pFfIxmrpdc-0/">Figma</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Key Feature 1]**    Application Delop on github.io
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://hagan199.github.io/)
```

Comment:
- [ ] This link is broken, it links to a `404` page (see image) kindly make sure your deployment is successful
![Screenshot from 2023-04-10 12-31-18](https://user-images.githubusercontent.com/91142432/230902829-b76f02ff-1f44-4f15-8cd4-92c540f1f3e7.png)

Reply:
Resolve it